lib/internal/cluster/child.js

```diff
@@ -99,7 +99,7 @@ function shared(message, handle, indexesKey, cb) {
     delete handles[key];
     delete indexes[indexesKey];
     return close.apply(this, arguments);
-  };
+  }.bind(handle);
```

Comment:
<blockquote>Bad value context for arguments value</blockquote>

See: https://gist.github.com/Hypercubed/89808f3051101a1a97f3